recipe/meta.yaml

```diff
@@ -13,21 +13,24 @@ source:
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - conda-pack = conda_pack.cli:main
 
 requirements:
   host:
     - python
     - pip
+    - wheel
```

Comment:
```suggestion
    - wheel
    - setuptools
```
conda-pack uses setuptools, see https://github.com/conda/conda-pack/blob/0.7.1/setup.py. Omitting it just happens to "magically" work for reasons that I won't go into details here.